src/shared/components/Banners/Banner/Banner.tsx

```diff
@@ -13,8 +13,8 @@ export const Banner = ({
   duration = 5000,
   onClose,
   hasCloseButton = !!onClose,
-  severity,
-  'data-testid': dataTestid,
+  severity = 'success',
+  'data-testid': dataTestid = `${severity}-banner`,
```

Comment:
just in case , we may have a several `success` or `error` banners with the same `data-testid`, right?

Reply:
Yes, that's possible, good callout. Perhaps this wasn't aware of the author of the JIRA task, as it may be hard to target a specific banner if multiple banner of the same type are shown. I'll seek clarification.